scriptmodules/helpers.sh

```diff
@@ -882,6 +882,20 @@ function setRetroArchCoreOption() {
     chown $user:$user "$configdir/all/retroarch-core-options.cfg"
 }
 
+## @fn setRetroArchConfigOption
+## @param option option to set
+## @param value value to set
+## @brief Sets a retroarch global config option in `$configdir/all/retroarch.cfg`
+function setRetroArchConfigOption() {
+    local option="$1"
+    local value="$2"
+    iniConfig " = " "\"" "$configdir/all/retroarch.cfg"
+    iniGet "$option"
+    if [[ -z "$ini_value" ]]; then
+        iniSet "$option" "$value"
+    fi
+}
+
```

Comment:
Not convinced about adding this function - also as it only sets a value if it isn't already set, the description/name isn't sufficient.

Reply:
It's similar to the options settings function (`setRetroArchCoreOption`), since there were 4 similar options needed to be set, I thought of using a function. We can re-use the previous option though, not a problem.

Reply:
Its not used outside of this module, so can do a hidden function in the module for it if you need to de-duplicate.

Reply:
sure, will modify with a local routine.

Reply:
Should be fixed now, I added a local routine in the scriptmodule script.